cogs/device.py

```diff
@@ -72,7 +72,7 @@ async def add_device(self, ctx: commands.Context) -> None:
                 descriptions = (
                     'Enter a name for your device.',
                     "Enter your device's identifier. This can be found with [AIDA64](https://apps.apple.com/app/apple-store/id979579523) under the `Device` section (as `Device String`).",
-                    "Enter your device's ECID (hex).",
+                    "Enter your device's ECID.",
```

Comment:
Keep the hex part of that string and just add decimal.

```suggestion
                    "Enter your device's ECID (hex or decimal).",
```

Reply:
why

Reply:
Users will ask for this if you remove it; just keep it.